recipes/nmslib/all/conanfile.py

```diff
@@ -57,13 +67,23 @@ def source(self):
 
     def generate(self):
         tc = CMakeToolchain(self)
+        tc.variables["WITH_EXTRAS"] = self.options.build_extras
+        if self.options.build_extras:
+            tc.preprocessor_definitions["WITH_EXTRAS"] = "1"
```

Comment:
I think that we can remove this one https://github.com/nmslib/nmslib/blob/master/similarity_search/CMakeLists.txt#L80 It seems that it's already adding the preprocessor definition